lib/workos/challenge_factor.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+# typed: false
+
+module WorkOS
+  # The ChallngeFactor class provides a lightweight wrapper around
+  # a WorkOS DirectoryUser resource. This class is not meant to be instantiated
+  # in DirectoryUser space, and is instantiated internally but exposed.
+  class ChallengeFactor
```

Comment:
This would be more consistent with what were calling these structures in the other SDKs.

```suggestion
  class Challenge
```

Reply:
done!